src/db/schema.ts

```diff
@@ -29,7 +29,8 @@ export const reviewRelations = relations(reviews, ({ one }) => ({
 
 export const userShows = pgTable('user_shows', {
   id: uuid().primaryKey().defaultRandom(),
-  hasReviewOrRating: boolean().notNull().default(false),
+  hasRating: boolean().notNull().default(false),
+  hasReview: boolean().notNull().default(false),
```

Comment:
seems like it'll be better in the future if there are separate flags for review vs rating, since a review when deleted will want to know if a rating is present